.github/workflows/vg-publish-package-template.yml

```diff
@@ -0,0 +1,84 @@
+name: publish package to npm registry
+on: 
+  workflow_call:
+    inputs:
+      is_pr_merged:
+        type: boolean
+        required: true
+    secrets:
+      VG_NODE_TOKEN:  #created by default in workflow
+        required: true
+
+jobs:
+  build_and_test:
+    name: Build and test job
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo '::set-output name=dir::$(yarn cache dir)'
+
+      - uses: actions/cache@v2
+        name: validate cache
+        id: yarn-cache
+        with:
+          path: | 
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Installing yarn dependencies
+        run: yarn install --frozen-lockfile --prefer-offline
+
+      - name: Lint
+        run: yarn lint
+
+      #- run: npm i
+      
+      # - run: yarn pack --filename testpkg.tgz
+      - run: yarn build
+      - name: Upload
+        uses: actions/upload-artifact@v2
+        with:
+            name: package
+            path: build/*
+      - name: Upload package
+        uses: actions/upload-artifact@v2
+        with:
+            name: packagejson
+            path: "package.json"
+
+  
+  publish:
+    if: ${{github.events.input.is_pr_merged}}=='True'

```

Comment:
is the value really 'True' or true?

Reply:
here 
https://github.com/actions/runner/issues/1483 both are cool

Reply:
but ill update it to true just for the sake of clarity